tempesta_fw/http.c

```diff
@@ -2693,7 +2694,8 @@ tfw_http_cli_error_resp_and_log(TfwHttpReq *req, int status, const char *msg,
 			spin_unlock(&cli_conn->seq_qlock);
 		}
 		else {
-			WARN_ONCE(list_empty_careful(&req->msg.seq_list),
+			WARN_ONCE(list_empty_careful(&req->msg.seq_list)
+				  && !test_bit(TFW_HTTP_B_REQ_DROP, req->flags),
 				  "Request is not in in seq_queue\n");
```

Comment:
Looks like this check can has false positives in any way: it can be called during the other thread is in `tfw_http_conn_cli_drop()` -  just between the sequence mentioned in [comment above](https://github.com/tempesta-tech/tempesta/pull/1193/files#r260360504).

Reply:
Yes, the only way to avoid false positives here is locking the seq_queue. Lock was added.